src/PassKit/PKPayLaterView.cs

```diff
@@ -0,0 +1,66 @@
+#nullable enable
+
+#if IOS && !__MACCATALYST__
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using ObjCRuntime;
+using Foundation;
+using PassKit;
+
+#if !NET
+using NativeHandle = System.IntPtr;
+#endif
+
+namespace PassKit {
+
+	public partial class PKPayLaterView {
+
+#if !NET
+		delegate void PKPayLaterValidateAmountCompletionHandler (IntPtr block, byte eligible);
+		static PKPayLaterValidateAmountCompletionHandler static_ValidateAmount = TrampolineValidateAmount;
+
+		[MonoPInvokeCallback (typeof (PKPayLaterValidateAmountCompletionHandler))]
+#else
+		[UnmanagedCallersOnly]
+#endif
+		static void TrampolineValidateAmount (IntPtr block, byte eligible)
+		{
+			var del = BlockLiteral.GetTarget<Action<bool>> (block);
+			if (del is not null) {
+				del (eligible != 0);
+			}
+		}
+
+#if NET
+		[SupportedOSPlatform ("ios17.0")]
+		[UnsupportedOSPlatform ("maccatalyst")]
+		[UnsupportedOSPlatform ("macos")]
+		[UnsupportedOSPlatform ("tvos")]
+#endif
+		[BindingImpl (BindingImplOptions.Optimizable)]
+		public void ValidateAmount (NSDecimalNumber amount, string currencyCode, Action<bool> callback)
```

Comment:
This can be a static method as far as I can tell:

```suggestion
		public static void ValidateAmount (NSDecimalNumber amount, string currencyCode, Action<bool> callback)
```

Reply:
Sounds good! On Xcode, the function is used as a static method so it makes sense to use it as such here!